tests/Feature/TeamInvitationEmailNormalizationTest.php

```
@@ -0,0 +1,63 @@
+        // Create a team
+        $team = Team::factory()->create();
+
+        // Create invitation with mixed case email
+        $invitation = TeamInvitation::create([
+            'team_id' => $team->id,
+            'uuid' => 'test-uuid-123',
+            'email' => 'Test@Example.com', // Mixed case
+            'role' => 'member',
+            'link' => 'https://example.com/invite/test-uuid-123',
+            'via' => 'link'
+        ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\TeamInvitation' in method 'test_team_invitation_normalizes_email_to_lowercase'. (undefined)

(StaticAccess)